e2e/test/scenarios/question/notebook-data-source.cy.spec.ts

```diff
@@ -272,6 +289,12 @@ describe("scenarios > notebook > data source", () => {
         assertDataPickerEntitySelected(1, "First collection");
         assertDataPickerEntitySelected(2, "Second collection");
         assertDataPickerEntitySelected(3, checkNotNull(modelDetails.name));
+
+        entityPickerModalTab("Recents").click();
+        cy.findByTestId("result-item")
+          .should("exist")
+          .and("contain.text", checkNotNull(modelDetails.name))
+          .and("have.css", "background-color", "rgb(80, 158, 227)");
```

Comment:
@kamilmielnik why do we want to assert css?

Reply:
Currently that's the only way to determine if a given item is selected or not.

Alternatively we could add `aria-selected` attribute. WDYT?

Reply:
not critial, but worth having a comment in the code

`aria-selected` or `aria-active` work for me

Reply:
Addressed here: #42825